x-pack/test/api_integration/apis/uptime/rest/telemetry_collectors.ts

```diff
@@ -84,6 +86,7 @@ export default function ({ getService }: FtrProviderContext) {
     after('unload heartbeat index', () => getService('esArchiver').unload('uptime/blank'));
 
     beforeEach(async () => {
+      await delay(1000);
```

Comment:
Is the delay still needed since we've removed the check for the API response and are now just checking the response code?

Reply:
removed.